client/src/Component/views/Methods/index.js

```diff
@@ -1,9 +1,13 @@
 import React from 'react';
+import Header from '../../Common/Header';
+import Footer from '../../Common/Footer';
 
 const Methods = () => {
   return (
     <div>
+      <Header titleText="Selecet your methods" />
```

Comment:
Also I hope this is okay to point this out but the prop should read "Select your methods" instead of "Selecet your methods". Just a tiny thing!